packages/marko-web/components/element/picture.marko

```diff
@@ -0,0 +1,25 @@
+import { getAsArray } from "@parameter1/base-cms-object-path";
+
+$ const hasImage = Boolean(input.image.src);
+
+<if(hasImage)>
+  <if(input.link)>
+    <marko-core-link ...input.link>
```

Comment:
Same here -- these attributes should be explicitly defined. Spreading them will bypass validation -- either remove the spread and pass the props explicitly, or remove the validation rules from the Marko component definition.